packages/amplify-e2e-tests/src/__tests__/auth_6.test.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  initJSProjectWithProfile,
+  getProjectMeta,
+  deleteProject,
+  amplifyPushAuth,
+  createNewProjectDir,
+  deleteProjectDir,
+  addAuthWithMaxOptions,
+} from 'amplify-e2e-core';
+import _ from 'lodash';
+
+const PROJECT_NAME = 'authTest';
+const defaultsSettings = {
+  name: PROJECT_NAME,
+};
+describe('zero config auth ', () => {
+  let projRoot: string;
+  beforeEach(async () => {
+    projRoot = await createNewProjectDir('zero-config-auth');
+  });
+
+  afterEach(async () => {
+    await deleteProject(projRoot);
+    deleteProjectDir(projRoot);
+  });
+
+  it('...should init a javascript project and add auth with a all uptions and update front end config', async () => {
+    await initJSProjectWithProfile(projRoot, defaultsSettings);
+    await addAuthWithMaxOptions(projRoot, { useInclusiveTerminology: false });
```

Comment:
`{ useInclusiveTerminology: false }` isn't needed for this helper function. It can be removed.

It's only used in `addAuthWithCustomTrigger()`.

Reply:
removed, thanks